### PR TITLE
blog: Making it easier to contribute to Servo

### PR DESCRIPTION
**Pull request by delan**

### CODE REVIEW

**Comment by delan** [comment]
The image doesn’t work in the preview, but it should work once deployed: https://servo.org/flaky-test-blog/img/blog/intermittent-dashboard.png

**Comment by mrego** [comment]
There's a plugin for footnotes if you want to give it a try: https://www.npmjs.com/package/eleventy-plugin-footnotes

File: _posts/2023-03-16-making-easier-to-contribute.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+## Intermittent dashboard
+
+To ensure that flaky tests can be discovered and fixed even if they are mitigated by retries, we’ve created an [intermittent dashboard](https://build.servo.org/intermittent-tracker/) that all unexpected results get reported to.
+
+Each result includes the test and subtest, the expected and actual outcomes, any test output, plus metadata like the commit and a link to the build. You can filter the data to a specific test or field value, and the dashboard automatically points out when all of the visible results have something in common, which can help us analyse the failures and identify patterns.
+
+For example, here we can see that all of the unexpected failures for one of the HTML parsing tests have the same assertion failure on the same subtest, but are not limited to one pull request:
+
+![screenshot of intermittent dashboard, filtered by test (/html/syntax/parsing/DOMContentLoaded-defer.html) and actual outcome (FAIL)]({{ site.url }}/img/blog/intermittent-dashboard.png)
```
Comment (mrego) [inline _posts/2023-03-16-making-easier-to-contribute.md:74]:
```suggestion
![screenshot of intermittent dashboard, filtered by test (/html/syntax/parsing/DOMContentLoaded-defer.html) and actual outcome (FAIL)]({{ '/img/blog/intermittent-dashboard.png' | url }})
```

This should fix the image in the preview.

**Comment by delan** [comment]
> There's a plugin for footnotes if you want to give it a try: https://www.npmjs.com/package/eleventy-plugin-footnotes

I tried this and thought it was a reasonable footnotes impl, but the [syntax for including markdown in footnotes](https://www.npmjs.com/package/eleventy-plugin-footnotes#user-content-example) is way too cumbersome imo. I don’t really want to spend time finding another footnotes plugin just for this post though, so maybe I’ll stick with the aside. Thanks anyway!

File: _posts/2023-03-16-making-easier-to-contribute.md (hunk 1 above)
Comment (delan) [reply _posts/2023-03-16-making-easier-to-contribute.md]: done in b8922d0e96a97d56105b54b797cb086a884fed0a

**Review by mrego — APPROVED** [review]
LGTM, thanks!

**Pull request merged**